lib/usb/usb_f103.c

```diff
@@ -89,16 +90,18 @@ static void usb_set_ep_rx_bufsize(usbd_device *dev, uint8_t ep, uint32_t size)
 {
 	(void)dev;
 	if (size > 62) {
-		if (size & 0x1f) {
-			size -= 32;
-		}
-		USB_SET_EP_RX_COUNT(ep, (size << 5) | 0x8000);
+		size += 31;	//round up to nearest mult. of 32
+		size >>= 5;
+		size -= 1;
+		size |= 0x20;	//set BL_SIZE bit
```

Comment:
no C++ comments please.  describe the algorithm, if necessary, in a block comment.